src/uct/cuda/Makefile.am

```diff
@@ -14,6 +14,11 @@ libuct_cuda_la_LDFLAGS  = $(CUDA_LDFLAGS) -version-info $(SOVERSION)
 libuct_cuda_la_LIBADD   = $(top_builddir)/src/ucs/libucs.la \
                           $(top_builddir)/src/uct/libuct.la \
                           $(CUDA_LIBS) $(NVML_LIBS)
+libuct_cuda_ladir       = $(includedir)/uct/cuda
+
+nobase_dist_libuct_cuda_la_HEADERS = \
+	cuda_ipc/cuda_ipc.cuh \
+	cuda_ipc/cuda_ipc_device.h
```

Comment:
I think `cuda_ipc/cuda_ipc_device.h` should be in `noinst_HEADERS`.

https://github.com/openucx/ucx/pull/10877

Reply:
Ah, no.
Now I see. It used in both places in `cuda_ipc_iface.c`, and in `cuda_ipc.cuh`...